packages/algob/src/lib/algo-operator.ts

```diff
@@ -210,8 +211,9 @@ export class AlgoOperatorImpl implements AlgoOperator {
     scParams: LogicSigArgs,
     flags: FundASCFlags,
     payFlags: TxParams,
-    txWriter: txWriter): Promise<LsigInfo> {
-    const lsig = await getLsig(name, scParams, this.algodClient);
+    txWriter: txWriter,
+    scInitParam?: Object): Promise<LsigInfo> {
```

Comment:
let's put this next to the `scParams` and please add docstring explaining all arguments.

Reply:
Ok, will have to move scParam next to scInitParam as scInitParam is optional and can't be put before required